.github/workflows/test_linux_jax_wheels.yml

```diff
@@ -0,0 +1,219 @@
+name: Test JAX  Wheels and Build Docker
+
+on:
+  workflow_call:
+    inputs:
+      amdgpu_family:
+        description: AMD GPU family directory in S3 (e.g., gfx94X-dcgpu)
+        required: true
+        type: string
+      release_type:
+        description: The type of release ("nightly" or "dev")
+        required: true
+        type: string
+      s3_subdir:
+        description: S3 subdirectory (e.g., v2 or v2-staging)
+        required: true
+        type: string
+      rocm_version:
+        description: ROCm version to use in docker build (e.g., 5.12)
+        required: true
+        type: string
+      tar_url:
+        description: URL to TheRock tarball to build against
+        required: true
+        type: string
+      python_versions:
+        description: Python version string used to compute CP tag (e.g., 3.12 -> cp312)
+        required: true
+        type: string
+      jax_ref:
+        description: Branch/tag/sha of rocm/rocm-jax to build
+        required: false
+        type: string
+        default: master
+      jax_test_branch:
+        description: JAX test repo branch to get unit tests from
+        required: false
+        type: string
+        default: "rocm-jaxlib-v0.7.1"
+      test_runs_on:
+        description: Runner label to use for testing (GPU-capable recommended)
+        required: false
+        type: string
+
+  workflow_dispatch:
+    inputs:
+      amdgpu_family:
+        type: choice
+        options:
+          - gfx110X-dgpu
+          - gfx1151
+          - gfx120X-all
+          - gfx94X-dcgpu
+          - gfx950-dcgpu
+        default: gfx94X-dcgpu
+      release_type:
+        type: choice
+        options: [dev, nightly]
+        default: dev
+      s3_subdir:
+        type: string
+        default: v2-staging
+      rocm_version:
+        type: string
+        required: true
+      tar_url:
+        type: string
+        required: true
+      python_versions:
+        type: string
+        required: true
+        default: "3.12"
+      jax_ref:
+        type: string
+        default: master
+      jax_test_branch:
+        type: string
+        description: JAX test repo branch to get unit tests from
+        default: "rocm-jaxlib-v0.7.1"
+      test_runs_on:
+        type: string
+        description: Runner label to use for testing (GPU-capable recommended)
+        default: "linux-mi325-1gpu-ossci-rocm"
```

Comment:
This input parameter is unused but probably shouldn't. One should build on CPU only but test an a GPU-equipped runner.